`'$this'` as `$sourcePropertyName` means complete source object (#60)
teamneusta/converter-bundle: docs/usage.md

```yaml
# config/services.yaml
services:
    YourNamespace\PersonNamePopulator: ~
```

Then declare the following converter in your package config:

```yaml
# config/packages/neusta_converter.yaml
neusta_converter:
    converter:
        person.converter:
            target: YourNamespace\Person
            populators:
                - YourNamespace\PersonNamePopulator

>                 # additional populators may follow

Reviewer: Please revert these unnecessary indentations in this file.

## Special Populators

> After working a while with the converter pattern, you will notice that many scenarios in the population are very
> similar.

Reviewer: Please don't add these strange line breaks.

```suggestion
After working a while with the converter pattern, you will notice that many scenarios in the population are very similar.
```

Be aware - that both properties have the same name should not lead you think they have the same type.
There is really an object conversion behind done by `address.converter`.

> If you set the `sourcePropertyName` as empty string the complete `source` object will be used for population.

Reviewer: ```suggestion
If you specify the `sourcePropertyName` as an empty string, the full `source` object is used for the population.
```

> Especially in addition wih the `ConvertingPopulator` this is sometimes necessary.

Reviewer: ```suggestion
Especially in connection with the `ConvertingPopulator` this is sometimes necessary.
```

But... do we really need this sentence? This whole paragraph is about the `ConvertingPopulator`.

Maybe we should rather say instead that this is only possible with the `PropertyMappingPopulator`.
